hs_core/templatetags/hydroshare_tags.py

```diff
@@ -44,11 +46,14 @@ def contact(content):
     if not content.is_authenticated():
         content = "Anonymous"
     elif content.first_name:
-        content = """<a href='/user/{uid}/'>{fn} {ln}<a>""".format(fn=content.first_name,
-                                                                   ln=content.last_name,
-                                                                   uid=content.pk)
+        content = format_html("<a href='/user/{uid}/'>{fn} {ln}</a>",
```

Comment:
User's first/last names will not be safely created as links to their profile using the |contact filter. This works with all of the |safes you made with |contact filters in the templates.